### PR TITLE
Fix #508 ,#395, #522,#304, blast-proof glass for 1.18, 1.18R1 rework, Simplified Chinese translation

### LINKED ISSUE #508: Arenas saved in slime format aren't reload after game ends
**Describe the bug**
Arenas saved in slime world format are unloaded and not reloaded after game ends, while normal worlds reloads as expected.

**To Reproduce**
1. Set up a arena and import it to slime world manager;
2. Play the arena and end the game;
3. Open game selection and you'll find the arena disappears;
4. Run `/swm list` and you'll see that the world is actually loaded, but not shown in arena selection, nor visible via `/bw arenaList`.

**Expected behavior**
The world of that arena should be loaded again

**System (please complete the following information):**
 - OS: Windows 11
 - Server software : PaperSpigot
 - Server Version : 1.18.1-216
 - Plugin Version : 22.3.4

**Additional context**
When I stopped my server, the log said this: 
```
[19:46:41 WARN]: [BedWars1058] Disabling arena: invasion
[19:46:41 ERROR]: Error occurred while disabling BedWars1058 v22.3.4 (Is it up to date?)
java.util.ConcurrentModificationException: null
        at java.util.LinkedList$ListItr.checkForComodification(LinkedList.java:970) ~[?:?]
        at java.util.LinkedList$ListItr.next(LinkedList.java:892) ~[?:?]
        at com.andrei1058.bedwars.BedWars.onDisable(BedWars.java:604) ~[bedwars-plugin-22.3.4.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:266) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:399) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:538) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.disablePlugins(SimplePluginManager.java:515) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.disablePlugins(CraftServer.java:496) ~[paper-1.18.1.jar:git-Paper-216]
        at net.minecraft.server.MinecraftServer.stopServer(MinecraftServer.java:1045) ~[paper-1.18.1.jar:git-Paper-216]
        at net.minecraft.server.dedicated.DedicatedServer.stopServer(DedicatedServer.java:800) ~[paper-1.18.1.jar:git-Paper-216]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1324) ~[paper-1.18.1.jar:git-Paper-216]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-216]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

(I apologize for my poor English since I'm not a native speaker)

### PR DESCRIPTION
**Pull request by hugeBlack**
Fix for #508
The onEnabled(a) call is somehow in the catch block, which can be hardly reached. So it is moved to be invoked after slime.generateWorld(finalWorld)

### CODE REVIEW

**Comment by hugeBlack** [comment]
I also added simplified Chinese translation.

**Comment by hugeBlack** [comment]
It seems that the getPlayerHead method is actually working properly in 1.18.

**Comment by hugeBlack** [comment]
Correctly set tnt igniter and add all stained glass to TNT whitelist for version 1.18_R1

**Comment by hugeBlack** [comment]
fix for #522 
Potion listener registration is somehow placed in the switch block for deprecated, so that the listener is only registered in these deprecated version

**Comment by hugeBlack** [comment]
The reason why noSuchMethod is thrown when isInvisibilityPotion method is called is that the isInvisibilityPotion method somehow vanished from VersionSupport and all version support classes in version 22.3.4, which is weird.

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/InvisibilityPotionListener.java [hunk 1]
```diff
@@ -74,9 +74,19 @@ public void onDrink(PlayerItemConsumeEvent e) {
...
                 5L);
-        //
 
-        if (nms.isInvisibilityPotion(e.getItem())) {
+        //The direct call of nms.isInvisibilityPotion somehow results in NoSuchMethod exception even though it actually exists, so reflection is used.
+        //TODO    Change this to direct call if you know why.
+        boolean isInvisibilityPotion = false;
+        try {
+            Class<?> vs = nms.getClass();
+            Method testPotionMethod = vs.getDeclaredMethod("isInvisibilityPotion",org.bukkit.inventory.ItemStack.class);
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/InvisibilityPotionListener.java:83]: why is reflection being used here?

File: resetadapter_slime/src/main/java/com/andrei1058/bedwars/arena/mapreset/slime/SlimeAdapter.java [hunk 2]
```diff
@@ -106,7 +106,11 @@ public void onEnable(IArena a) {
 
                 // This method must be called synchronously
                 SlimeWorld finalWorld = world;
-                Bukkit.getScheduler().runTask(getOwner(), () -> slime.generateWorld(finalWorld));
+                Bukkit.getScheduler().runTask(getOwner(), () -> {
+                    slime.generateWorld(finalWorld);
+                    onEnable(a);
```
Comment (andrei1058) [inline resetadapter_slime/src/main/java/com/andrei1058/bedwars/arena/mapreset/slime/SlimeAdapter.java:111]: this is wrong. why you making it recursive in here?

**Review by andrei1058 — CHANGES_REQUESTED** [review]

**Comment by andrei1058** [comment]
please resolve conflicts

**Pull request closed without merging**